app/assess/models/round.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+from typing import List
+
+from slugify import slugify
+
+from .application import Application
+
+
+class Round(object):
```

Comment:
Nice data models! 

You can deduce boilerplate by using data classes (in the standard library) or attrs (a data class library which is used by basically 90% of python people, me included). 

see this [video](https://www.youtube.com/watch?v=vBH6GRJ1REM)

Reply:
Great video, on it

Reply:
Dataclasses done